extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -609,8 +606,18 @@ export class AmpA4A extends AMP.BaseElement {
           // Load any extensions; do not wait on their promises as this
           // is just to prefetch.
           const extensions = extensionsFor(this.win);
-          creativeMetaDataDef.customElementExtensions.forEach(
-            extensionId => extensions.loadExtension(extensionId));
+          creativeMetaDataDef.extensions.forEach(
+            extension => {
+              const extensionId = extension['custom-element'];
+              const extensionSrc = extension['src'];
+              const versionStartIndex = extensionSrc.indexOf(extensionId) +
+                  extensionId.length + 1;
+              // -3 because the src string always ends with '.js'.
+              const versionEndIndex = extensionSrc.length - 3;
+              const extensionVer = extensionSrc.substring(
+                  versionStartIndex, versionEndIndex);
+              extensions.loadExtension(extensionId, extensionVer);
```

Comment:
would it be the case that we want to load same extension with different extension version?
for example `loadExtensions('amp-vine', 0.1)` `loadExtensions('amp-vine', 0.2)`. I don't think it will work.